biorxiv/article_distances/03_biorxiv_corrected_published_rate.py

```diff
@@ -292,3 +292,82 @@
 g.save("output/figures/publication_rate_rerun.svg")
 g.save("output/figures/publication_rate_rerun.png", dpi=250)
 print(g)
+
+# # Plot Publication Rate
+
+# +
+publish_rate_df["pub_month"] = pd.Categorical(
+    publish_rate_df.pub_month.values.tolist(), ordered=True
+)
+
+posted = (
+    publish_rate_df.query("label=='2020 Snapshot+Missing Links'")
+    .query("pub_month < '2019-01'")
+    .posted.sum()
+)
+
+published = (
+    publish_rate_df.query("label=='2020 Snapshot+Missing Links'")
+    .query("pub_month < '2019-01'")
+    .published.sum()
+)
+print(f"Published: {published}")
+print(f"Posted: {posted}")
+print(f"Overall proportion published: {published/posted:.4f}")
+# -
+
+color_mapper = {
+    "2018": "#a6cee3",
+    "2020ML": "#33a02c",
+    "2020": "#1f78b4",
+}
+
+g = (
```

Comment:
I would add a title so that you know which one truncated. The trends look the same as expected, but the overall publications is higher after your truncate since 2019 will include many papers that are not published yet, is that right?

Reply:
Is there a reason you still see 2019 in the truncated figure?

Reply:
>I would add a title so that you know which one truncated. The trends look the same as expected, but the overall publications is higher after your truncate since 2019 will include many papers that are not published yet, is that right?

Yup. You are correct. One of my reviewers wanted to see how the rate looked if you didn't take 2019+ into account. Hence my cutoff plot.

>Is there a reason you still see 2019 in the truncated figure?

To show the future of preprints being posted onto bioRxiv. I didn't want to remove that part of the figure as its removal would open a door for a follow up request asking about 2019-2020 posted rate.